sdk/storage/azure-storage-blob/tests/test_blob_client.py

```diff
@@ -559,5 +559,17 @@ def callback(response):
         custom_headers = {'User-Agent': 'customer_user_agent'}
         service.get_service_properties(raw_response_hook=callback, headers=custom_headers)
 
+    @GlobalStorageAccountPreparer()
```

Comment:
You don't need an account preparer for this test. It's only testing the constructor.